test/org/sagebionetworks/bridge/services/SurveyServiceMockTest.java

```diff
@@ -98,7 +108,14 @@ public void successfulDelete() {
         doReturn(survey).when(mockSurveyDao).getSurvey(any());
         
         service.deleteSurvey(survey);
-        
+
+        // verify query args
+        ArgumentCaptor<String> queryCaptor = ArgumentCaptor.forClass(String.class);
+        verify(mockSharedModuleMetadataService).queryAllMetadata(anyBoolean(), anyBoolean(), queryCaptor.capture(),  anySetOf(String.class));
```

Comment:
We want to verify all the query args, not just the where clause. Similarly in successfulDeletePermanently().